xml/Microsoft.Extensions.FileProviders.Internal/PhysicalDirectoryContents.xml

```diff
@@ -109,7 +109,8 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>
+          <see langword="true" /> if a directory was located at the given path.</summary>
         <value>To be added.</value>
```

Comment:
This information should be split between the summary and the value:
```suggestion
        <summary>Determines if a directory was located at the given path or not.</summary>
        <value><see langword="true" /> if a directory was located at the given path; <see langword="false" /> otherwise.</value>
```